inc/config.class.php

```diff
@@ -2541,6 +2547,10 @@ static function checkWriteAccessToDirs($fordebug = false) {
                      __s('Web access to files directory is protected')."\"></td></tr>";
             }
          }
+      } else {
+            echo "<td><img src='".$CFG_GLPI['root_doc']."/pics/warning_min.png'>".
+                  "<p class='red'>".__('Web access to the files directory, should not be allowed')."<br/>".
+                  __('Automatic checks cannot be done; please review .htaccess file and the web server configuration.')."</p></td></tr>";
       }
```

Comment:
IIRC, there is tons of false positive... btw, ok to raise admin attention

Reply:
I totaly agree... But this will permit to get rid of "files directory is insecure". We all agree that the way to go is to store files outside the webroot ;)